### ISSUE TITLE
Download Page for the Library

### ISSUE THREAD

**Issue by lordzden** [jfeinstein10/SlidingMenu#460]
Where can I download the library? By the time that I try to copy the project I got a 100 errors. I really wanted to see how the Responsive UI works... Thanks

Description Resource    Path    Location    Type
[Accessibility] Missing contentDescription attribute on image   country_listitem.xml    /Main/res/layout    line 8  Android Lint Problem
[I18N] Hardcoded string "Country Name", should use @string resource country_listitem.xml    /Main/res/layout    line 16 Android Lint Problem
[I18N] Hardcoded string "Country Abbrev", should use @string resource   country_listitem.xml    /Main/res/layout    line 24 Android Lint Problem
The method transformCanvas(Canvas, float) of type new CanvasTransformer(){} must override or implement a supertype method   CustomSlideAnimation.java   /jeremyfeinstein/src/com/jeremyfeinstein/slidingmenu/example/anim   line 25 Java Problem
.
.
.
